src/grecon/grecon.c

```diff
@@ -103,7 +103,7 @@ void grecon(struct grecon_conf* param,  const long dims1[DIMS], complex float* o
 		sens1 = cov1;
 	}
 
-	fftmod(DIMS, dims1, FFT_FLAGS, sens1, sens1);
```

Comment:
using cov1_dims is incorrect, but using dims1 is also incorrect for the case where the sens1 does have higher order dimensions but the dims1 does. I have added a sens_dims that checks for the correct size with and without caltwo enabled